CMakeLists.txt

```diff
@@ -15,14 +15,15 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
 # Using Cmake version 3.11 for better CUDA & Fortran support
-cmake_minimum_required(VERSION 3.11)
-
-# location for CMake modules (MDI)
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+cmake_minimum_required(VERSION 3.11 FATAL_ERROR)
 
 # Specify project name & programming language (C is only needed for auto-detection of MKL by find_package)
 project(openmopac LANGUAGES Fortran C)
```

Comment:
Shouldn't C be optional i.e. only if MKL is used?

Reply:
Yes, if I set things up so that there is prior knowledge of whether MKL will be used or not. I'd like to have a fail-safe default, so that people who don't know what they are doing have a chance of automatically detecting MKL with find_package (when its "magic" works correctly). A reasonable possibility would be to add an MKL option that defaults to ON and drops the C requirement if it is set to OFF.